CI/.drone.yml

```diff
@@ -38,6 +38,6 @@ steps:
   # generate test scripts
   - /bin/bash bin/tests/run-all-test
   # generate all petstore samples (client, servers, doc)
-  - /bin/bash bin/run-all-petstore
+  - /bin/bash bin/generate-samples.sh
   # generate all petstore samples (openapi3)
-  - /bin/bash bin/openapi3/run-all-petstore
+  - /bin/bash bin/generate-samples.sh bin/configs/other/*.yaml
```

Comment:
So this other folder naming is still confusing to me
This comment says that ll samples in the other folder are v3 samples
looking in them I see v3 samples. Is this comment incorrect?
Are these v2 or v3?
Are they non-petstore samples?
Can you clarify this in the comment or the folder name?

Reply:
Everything will move to openapi3 shortly, so this comment will only be stale for a brief time.

Reply:
So these are v2 samples only then?

Reply:
No. As mentioned in the PR, these are the samples which were not previously included in our ensure script.

Any v2 samples in this directory won't be v2 for long, so the incorrectness of the comment is only temporary.

We don't really need to be flipping back and forth between words in comments to have them be correct across 20 or so commits (however many it'll take for conversion). There's a ton more that we can focus our energies on.

Plus, if CI is already running or queued, making a change to a few comments wastes the time of authors and reviewers. It's also bad for the environment if we're churning out unnecessary builds over and over.

Reply:
I understand that changing a comment kicks off a new CI run, and that doing a ton of them is not good. One of the points that you highlighted was that before, people were updating incorrect locations for generators. Some reasons that they were doing this was that multiple locations were possible and also because they misunderstood what to do. My question is looking to clear up any misunderstanding that users have when trying to understand what the other folder is for. In my opinion kicking off one more CI run is a very low cost to clearing up confusion. But it's fine if you want to keep it as is.

Reply:
I reviewed and I think I only mentioned this in chat, so I'll reproduce it here…

I'd personally like to include all these yaml in a single directory. It only takes about a minute on my machine to generate all generators.

Reply:
A single directory is clearer longer term. Keeping it separate for now is fine. This PR has been in review for a while and this item isn't a blocking concern to me.